themes/bootstrap5/templates/Recommend/Databases.phtml

```diff
@@ -1,32 +1,35 @@
+<?php
+  $groupedDatabases = $this->recommend->getGroupedResults();
+  if (!$groupedDatabases) {
+    $databases = $this->recommend->getResults();
+  }
```

Comment:
If you take my suggestion above, then this could become:


```suggestion
  if ($this->recommend->useGroupedResults()) {
    $groupedDatabases = $this->recommend->getGroupedResults();
  } else {
    $databases = $this->recommend->getResults();
  }
```